CHANGELOG.md

```diff
@@ -28,6 +28,8 @@ at anytime.
   * daemon not erring properly for non-numeric values being passed to the `bid` parameter for the `publish` method
   * `publish` command to allow updating claims with a `bid` amount higher than the wallet balance, so long as the amount is less than the wallet balance plus the bid amount of the claim being updated (https://github.com/lbryio/lbry/issues/748)
   * incorrect `blob_num` for the stream terminator blob, which would result in creating invalid streams. Such invalid streams are detected on startup and are automatically removed (https://github.com/lbryio/lbry/issues/1124)
+  * fixed the inconsistencies in docstrings
```

Comment:
inconsistencies in API docstrings

Reply:
Ok, but it is valid for both API and CLI
Will the following be better?
inconsistencies in API and CLI docstrings